web/src/stories/icon.stories.mdx

```diff
@@ -64,6 +92,12 @@ i := component.NewIcon("user", component.WithTooltip("hello", component.TooltipM
 i.Config.Color = "#2596be"
 `}}
 
+export const iconWithSvgDocs = {source: {code: `
+i := component.NewIcon("my-custom-icon")
```

Comment:
There is potential for a name collision with clarity icons (https://clarity.design/foundation/icons/shapes/) that results in showing an empty svg. Please document this behavior or use the client to render an error in such a case.